pacemaker.spec.in

```diff
@@ -169,7 +170,7 @@ It will run scripts at initialization, when machines go up or down,
 when related resources fail and can be configured to periodically check
 resource health.
 
-Available rpmbuild rebuild options:
+Available rpmbuild rebuild options (not necessarily applicable/with effect):
```

Comment:
Unfortunately, even if the first condition is satisifief, passing `--with-cman` to `configure` will *not* fail on unsatisfied prerequisites :-/
The conclusion is that explicit user requirements are not obeyed and the validity of the added note remains (there may be better wording, though).